src_Core/Near_Mem_VM/Near_Mem_Caches.bsv

```diff
@@ -209,7 +209,7 @@ module mkNear_Mem (Near_Mem_IFC);
 
       // CPU side: DMem response
       method Bool       valid      = dcache.valid;
-      method Bit #(64)  word64     = dcache.word64;
+      method Bit #(64)  word64     = dcache.cword;
```

Comment:
I wonder whether this, `st_amo_val` and the `store_value` arguments to `req` should become `CWord` too or not. Both are valid design decisions. Though for our use-case it would be nicer to push it out to this interface.

Reply:
Though maybe not, given we'd only add one tag here, whereas there could be multiple tags per CWord. It'd be nice if this were called something other than `word64` though at least, say just `data`, so we only need to change the type.